TelemetryManager/Inc/c_library_v2/cubepilot/cubepilot.h

```diff
@@ -0,0 +1,70 @@
+/** @file
+ *  @brief MAVLink comm protocol generated from cubepilot.xml
+ *  @see http://mavlink.org
```

Comment:
Do we need the entire library or can we make do with a subset? for example do we need this cubepilot folder?

Reply:
Good point. Will look into it. Likely not.

Reply:
I have removed the unnecessary folders from the library.